res/xml/reddit_preferences.xml

```diff
@@ -145,4 +145,9 @@
         android:defaultValue="true"
         android:title="@string/pref_load_javascript" />
 
+    <CheckBoxPreference
+        android:key="imgur_direct"
+        android:defaultValue="false"
```

Comment:
You're welcome to make this true if you'd prefer.  On reconsideration as long as people can turn it off it's no big.